include/tvm/topi/detail/broadcast.h

```diff
@@ -51,8 +51,8 @@ inline BroadcastHelper BroadcastShape(const tvm::Array<tvm::PrimExpr>& shape1,
   int i;
   for (i = 1; i <= std::min(s1_size, s2_size); ++i) {
     // TODO(@icemelon9): Need to revisit this part
-    const VarNode* var1 = shape1[s1_size - i].as<VarNode>();
-    const VarNode* var2 = shape2[s2_size - i].as<VarNode>();
+    const IntImmNode* static_size1 = shape1[s1_size - i].as<IntImmNode>();
```

Comment:
BTW, any difference in checking Any or int?

Reply:
We need to check IntImm to include any symbolic expression type. Here Any might not work.